src/main/java/io/specto/hoverfly/junit/dsl/RequestTemplateBuilder.java

```diff
@@ -26,13 +26,13 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import static io.specto.hoverfly.junit.core.model.FieldMatcher.fromString;
+import static io.specto.hoverfly.junit.dsl.matchers.ExactMatcher.exactlyMatches;
 import static org.apache.commons.lang3.CharEncoding.UTF_8;
 
 /**
- * A builder for {@link RequestMatcher}
+ * A builder for {@link Request}
  */
-public class RequestMatcherBuilder {
+public class RequestTemplateBuilder {
```

Comment:
Template type was poorly named as what was taking place was not templating, it was matching. We match requests to matchers. Templating would be when we substitute variables in a response body for variables. If we muddle the two terms it will get confusing when actual templating is implemented. I one is package private and one is public, it's not too confusing to share names. We could also stick `DTO` or something at the end of these ones.